tensorflow/core/distributed_runtime/session_mgr.cc

```diff
@@ -76,7 +76,14 @@ Status SessionMgr::CreateSession(const string& session,
 
   std::shared_ptr<WorkerSession> worker_session;
 
-  if (isolate_session_state) {
+  if (isolate_session_state || server_def.cluster().job_size()) {
+    if (server_def.cluster().job_size()) {
+      LOG(INFO) << "ClusterSpec propagation is enabled.";
```

Comment:
In general, our users don't like when log messages are printed out that they can't control. Could this be turned to a `VLOG(1)`?

Reply:
The code is updated now. Thanks a lot!